src/modules/test/recipes/SampleTests.json

```diff
@@ -6,13 +6,14 @@
         "ExpectedResult": 0,
         "WaitSeconds": 0
     },
-    {
-        "ComponentName": "SampleComponent",
-        "ObjectName": "reportedIntegerObject",
-        "Desired": false,
-        "ExpectedResult": 0,
-        "WaitSeconds": 0
-    },
+    // TODO: Enable this test once the bug is fixed
```

Comment:
Comments are invalid JSON. Looks like the tests are passing, but are we sure there are no other unintended consequences?

Reply:
although its invalid, we ignore the comments when parsing, doesn't break it